### ISSUE TITLE
learning rate adjust

### ISSUE THREAD

**Issue by YuHang007** [G-U-N/PyCIL#97]
if I want to increase the batch size, how to modify the lr.

**Comment by zhoudw-zdw** [comment]
https://github.com/G-U-N/PyCIL/blob/87cc4110d60765dc3f64af7ef7a693efed8418ff/models/finetune.py#L16

It is simple.
